streams/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -489,11 +489,12 @@ class ZStreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
 
   private def fromInputStream = unsafeRun {
```

Comment:
Thanks for enhancing the test! Let's stick with either a property-based test or a deterministic test with constants. That's okay too.

The problem with using `Random` directly is that failures would not be easily reproducible.

Reply:
Yeah, you are totally right, I'll go with a fixed seeded Random if you are fine with it. BTW I am new to the workflow: Just amend the change and force push to my forked repo?

Reply:
Feel free to amend or add new commits. We'll squash it anyway.

Would it be possible to avoid the use of Random entirely? Not sure it's necessary to demonstrate the issue here, and for larger coverage we should use property checks.

Reply:
Of course, will do so. It's only important to use more than one chunk and  data that is not exactly dividable into full chunks. Thanks for your quick feedback and help.

Reply:
🙏🏻